docs/source/commandline/commandline-all.rst

```diff
@@ -496,137 +680,373 @@ Core rotation options
 
+.. _mame-commandline-pausebrightness:
+
 **-pause_brightness** *<value>*
 
 	This controls the brightness level when MAME is paused. The default value is *0.65*.
 
+.. _mame-commandline-effect:
+
 **-effect** *<filename>*
 
 	Specifies a single PNG file that is used as an overlay over any game screens in the video display. This PNG file is assumed to live in the root of one of the artpath directories. The pattern in the PNG file is repeated both horizontally and vertically to cover the entire game screen areas (but not any external artwork), and is rendered at the target resolution of the game image. For -video gdi and -video d3d modes, this means that one pixel in the PNG will map to one pixel on your output display. The RGB values of each pixel in the PNG are multiplied against the RGB values of the target screen. The default is '*none*', meaning no effect.
 
 
 
-Core vector options
+Core Vector Options
 -------------------
 
+.. _mame-commandline-noantialias:
+
 **-[no]antialias** / **-[no]aa**
 
 	Enables antialiased line rendering for vector games. The default is ON (*-antialias*).
 
+.. _mame-commandline-beam:
+
 **-beam** *<width>*
 
 	Sets the width of the vectors. This is a scaling factor against the standard vector width. A value of 1.0 will keep the default vector line width. Smaller values will reduce the width, and larger values will increase the width. The default is *1.0*.
 
+.. _mame-commandline-flicker:
+
 **-flicker** *<value>*
 
 	Simulates a vector "flicker" effect, similar to a vector monitor that needs adjustment. This option requires a float argument in the range of 0.00 - 100.00 (0=none, 100=maximum). The default is *0*.
 
 
 
-Core sound options
+Core Sound Options
 ------------------
 
+.. _mame-commandline-samplerate:
+
 **-samplerate** *<value>* / **-sr** *<value>*
 
 	Sets the audio sample rate. Smaller values (e.g. 11025) cause lower audio quality but faster emulation speed. Higher values (e.g. 48000) cause higher audio quality but slower emulation speed. The default is *48000*.
 
+.. _mame-commandline-nosamples:
+
 **-[no]samples**
 
 	Use samples if available. The default is ON (*-samples*).
 
+.. _mame-commandline-volume:
+
 **-volume** / **-vol** *<value>*
 
 	Sets the startup volume. It can later be changed with the user interface (see Keys section). The volume is an attenuation in dB: e.g., "**-volume -12**" will start with -12dB attenuation. The default is *0*.
 
+.. _mame-commandline-sound:
+
+**-sound** *<dsound|sdl|none>*
```

Comment:
There are also `coreaudio` (Mac only), `xaudio` (Windows only) and `portaudio` options.  The `portaudio` option supports a variety of drivers and is generally the lowest latency option on Windows and Linux, while on Mac `coreaudio` is the default and generally provides lowest latency and overhead.

Reply:
Fixed on my side.